### DISCUSSION TITLE
Anyone running Masa on Coldfusion 2023 Standard?

### DISCUSSION THREAD

**Discussion by no3rdw** [MasaCMS/MasaCMS#196] [General]
I'm preparing to move from CF 2016 Enterprise to CF 2023, and plan to "downgrade" to Standard as I don't really use any of the features available only in Enterprise.  

I am wondering if anyone has Masa running on CF 2023 Standard and if you've encountered any issues or limitations.  Thanks!

**Comment by ptutty** [comment]
I've got it running on CF2021 enterprise and developer edition. Also on CF2018 which is EOL soon.

**Comment by guustnieuwenhuis** [comment]
Over the last few day, I've been trying to run Masa CMS with Adobe ColdFusion 2023 **on Docker**. Unfortunately it seems that I might have discovered a serious bug with Adobe ColdFusion that makes it impossible to run Masa CMS in this configuration.

The bug has been reported to Adobe, I'm waiting for their confirmation. Once they confirm my finding, I'll share the details.

I haven't tested on non-Docker environments.
